storagenode/contact/chore.go

```diff
@@ -77,6 +77,9 @@ func (chore *Chore) Run(ctx context.Context) (err error) {
 			interval := initialBackOff
 			attempts := 0
 			for {
+
+				mon.Counter("satellite.request").Inc(1)
```

Comment:
You can add `//locked` to the end of this line and then run ` go generate ./scripts/check-monitoring.go` to add the stat to the `monkit.lock` file.

Reply:
Also, I would suggest `satellite_request` instead of `satellite.request` since we have generally used underscores in other metric names.

Reply:
Change done!  :)

Reply:
For future reference, you can click the refresh button next to a name in the reviewers panel to re-request someone's review, and they will be notified. Usually I will wait for that instead of comment replies.